index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" type="text/css" href="bower_components/bootstrap/dist/css/bootstrap.css">
+
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+
+  <title>Cars</title>
+</head>
+<body>
+<nav class="navbar navbar-default navbar-fixed-top">
+  <div class="container-fluid">
+       <form class="navbar-form navbar-left">
+        <div class="form-group">
+          <input type="text" class="form-control" placeholder="Enter Description" oninput="CarLot.activateEvenMoreEvents(event);" onchange="CarLot.activateMoreEvents(event);" id="searchBar">
+        </div>
+        <button type="submit" class="btn btn-default" id="submitButton">Submit</button>
+      </form>
+  </div>
+</nav>
+
+<div class="container">
+  <div class="row">  <!-- row 1 -->
```

Comment:
I think I missed looking at your html file before. So these `<div>`s should be dynamically created in your javascript file.